docker-compose.override.yml

```diff
@@ -6,34 +6,30 @@ services:
     container_name: lp-temporal-postgresql
   temporal:
     container_name: lp-temporal
-    environment:
-      - PROMETHEUS_ENDPOINT=0.0.0.0:9090
   temporal-admin-tools:
     container_name: lp-temporal-admin-tools
   temporal-web:
     container_name: lp-temporal-web
   prometheus:
     container_name: lp-prometheus
-    image: prom/prometheus
+    build: './deployment/prometheus'
+    depends_on:
+      - temporal
     networks:
       - temporal-network
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
```

Comment:
Why is this needed?

Reply:
prob left over from testing will try without

Reply:
fixed